.gitignore

```diff
@@ -16,5 +16,4 @@ config.status
 src/coreSig.ml
 src/extSigs.ml
```

Comment:
You can remove those two now they are not generated

Reply:
Ok